.github/workflows/ci.yml

```diff
@@ -227,9 +227,32 @@ jobs:
       - name: Rename Binary with Target Name
         run: mv target/${{ matrix.target }}/release/tailcall target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}
 
+      # Copy the CLI binary and rename it to include the name of the target platform
+      - name: Copy CLI binary
+        id: copy-cli
+        run: |
+          mkdir dist
+          cp target/${{ matrix.target }}/release/tailcall ./dist/tailcall-${{ matrix.target }}
+
       - name: Upload ${{ matrix.target }} Binary
         uses: xresloader/upload-to-github-release@v1
         with:
           release_id: ${{ needs.draft_release.outputs.create_release_id }}
           file: target/${{ matrix.target }}/release/tailcall-${{ matrix.target }}
           overwrite: true
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Generate npm packages
+        run: cd npm/tailcall && npm run gen
+        env: APP_VERSION=${{ needs.draft_release.outputs.create_release_name }}
+
+      - name: Publish npm packages as latest
```

Comment:
the updated files won't be committed to the repo and the repo content will be different from the published one on the npm.

I don't know if this is much a problem

Reply:
No it won't be a problem.